packages/ui/components/InputCombobox/index.tsx

```
@@ -0,0 +1,328 @@
+                          className="mr-2 shrink-0"
+                        />
+                      )}
+                      <span className="truncate">{option.label}</span>
+                    </CommandItem>
+                  ))}
+                </CommandGroup>
+              )}
+              {groupedOptions?.map((group, idx) => (
+                <>
+                  {idx !== 0 && <CommandSeparator key={idx} className="bg-slate-300" />}
+                  <CommandGroup heading={group.label}>
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)